include/net/golioth/lightdb.h

```diff
@@ -158,6 +158,46 @@ int golioth_lightdb_delete_cb(struct golioth_client *client, const uint8_t *path
  */
 int golioth_lightdb_delete(struct golioth_client *client, const uint8_t *path);
 
+/**
+ * @brief Push value to Golioth's LightDB Stream (callback based)
+ *
+ * Asynchronously push new value to LightDB Stream and let @p cb be invoked when server
+ * acknowledges it or some error condition happens.
+ *
+ * @param client Client instance
+ * @param path LightDB Stream resource path
+ * @param format Format of payload
+ * @param data Payload data
+ * @param data_len Payload length
+ * @param[in] cb Callback executed on response received, timeout or error
+ * @param[in] user_data User data passed to @p cb
```

Comment:
Inconsistent usage of param in/out annotations.

Reply:
Right, will fix that. While we are on that topic, what should be the in/out annotation for `struct golioth_client *client` argument? `@param[in]` or `@param[in,out]`?

Reply:
Updated annotations in `lightdb.h`.